doc/collapse.md

```diff
@@ -68,3 +79,17 @@ Woltka supports collapsing a [stratified](stratify.md) profile using one field i
 ### Feature names
 
 Once a profile is collapsed, the metadata of the source features ("Name", "Rank", and "Lineage") will be discarded. One may choose to supply a target feature name file by `--names` or `-n`, which will instruct the program to append names to the profile as a metadata column ("Name").
+
+
+## Considerations
+
+It is important to highlight that one-to-many mapping may change some of the
+underlying statistical assumptions of downstream analyses.
+
+In the default mode, because one source may be collapsed into multiple targets, the total feature count per sample may be inflated, and the relative abundance of each feature may no longer correspond to that of the sequences assigned to it. In another word, this breaks the [compositionality](https://en.wikipedia.org/wiki/Compositional_data) of the data.
+
+How significantly this may impact an analysis depends on the relative frequency of multiple mappings found in the data, the biological relevance of the affected features, and the statistical nature of the analysis.
+
+For example, in the `reaction-to-ec.txt` file under [MetaCyc](metacyc.md), 80 out of 3618 (2.2%) reactions have more than one corresponding EC number. Whether such a translation may be considered as unique (and whether the resulting table is still compositional) is a call of the user.
+
+A solution to this is to turn on the [division](#division) flag (`-d`). This guarantees that the sum of feature counts remains the same after collapsing. But one should consider the biological implication before making a decision (see [above](#division)).
```

Comment:
Could we report the # of multiple mappings during analysis? humann3 does this when you regroup the table from gene families to other classification systems. I'm not sure how relevant it would be, but it is a consideration itself.

Reply:
Good point! I didn't know that HUMAnN3 does that. Can you point to their documentation? Maybe it's worth to have an issue and consider implementation in the future.

Reply:
Link to the collapsing as discussed [here](https://github.com/biobakery/humann#humann_regroup_table)
Source code [here](https://github.com/biobakery/humann/blob/b5e1485342f25ed688fe6d22029cd3cc05d0d576/humann/tools/regroup_table.py#L190)

When you regroup the table, you get a final report on screen stating the # of times a feature was grouped more than once, and more than twice.

Reply:
Thank you! This seems neat, and shouldn't be hard to implement.